### ISSUE TITLE
Earthly Remote Buildkit Configuration - config.yaml

### ISSUE THREAD

**Issue by MatthiasScholzTW** [earthly/earthly#1177]
Just a simple questions since the documentation is lacking on a specific point and some trial and error did not help.

# Problem Statement

WORKING: Using this command allows to connect to the external earthly buildkit server:
`EARTHLY_BUILDKIT_HOST=tcp://<server_dns_name>:8372 earthly +<step>`

NOT WORKING: Trying to make this happen in the configuration `~/.earthly/config.yml` was not successful.
This is how it currently looks like :
```
global:
    earthly_buildkit_host: tcp://<server_dns_name>:8372

# or
global:
   buildkit_host: tcp://<server_dns_name>:8372
```
NOTE: This was set using `earthly config global.buildkit_host tcp://<server_dns_name>:8372`.

# Question

Most probably the key in `config.yml` is just incomplete, but I can not figure out how it should look like.
How should the parameter be properly configured? What would have been the place to find it myself?

# Documentation Checked

- https://hub.docker.com/r/earthly/buildkitd

> Assuming you ran this on another machine named fast-builder, you could use this remote buildkitd by setting EARTHLY_BUILDKIT_HOST=tcp://fast-builder:8372, **or by specifying the address in your config.yml**.

- https://docs.earthly.dev/ci-integration/remote-buildkit

> `buildkit_host`

- https://docs.earthly.dev/docs/earthly-config

**Comment by vladaionescu** [comment]
Hi @MatthiasScholzTW - in your example config I see `earthly_buildkit_host`. I believe the correct key is simply `buildkit_host`. But maybe you have tried both?

CC @dchw

**Comment by MatthiasScholzTW** [comment]
Indeed, I gave both options a try.  With and without `earthly_` prefixed.

**Comment by vladaionescu** [comment]
As far as I can tell you did everything right. It's probably a bug.

**Comment by dchw** [comment]
Should be fixed with #1180.

The TL;DR is that the environment variable worked along an old codepath, while the setting in the file would only be activated by _another_ setting intended to feature gate the `buildkit_host` setting. This PR should remove that feature flag and fix this case.

**Comment by MatthiasScholzTW** [comment]
Can confirm the latest release: `v0.5.23` solve the issue.

For the record the configuration option is:
`earthly config global.buildkit_host tcp://<dns_name_of_build_server>:8372`

Thanks for the quick fix!
